src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -9,14 +9,16 @@ namespace System.Diagnostics
 {
     public partial class Process
     {
+        private const int NanoSecondsTo100NanoSecondsFactor = 100;
```

Comment:
Nit, `Nanoseconds` not `NanoSeconds` (I see the error is copied in FreeBSD\Interop.Process.cs.)

Since `nanosecond` is one word

Reply:
Fixed it. But now there are 2 different spellings... Since these are private fields I think it's fine this way - or we also fix it in `FreeBSD\Interop.Process.cs`... your call.

Reply:
if youre offering yes please let's fix both

Reply:
👍 will open a tiny PR with that (not mixing with this one, since it's unrelated).